mssqlcli/mssql_cli.py

```diff
@@ -299,6 +289,7 @@ def handle_editor_command(self, text):
         # FIXME: using application.pre_run_callables like this here is not the best solution.
         # It's internal api of prompt_toolkit that may change. This was added to fix #668.
         # We may find a better way to do it in the future.
+        # pylint: disable=no-member
```

Comment:
So this disabling is related to the FIXME comment? If we solve, the linting issue would likely go away?

Reply:
Yes. I created an issue to refactor the `special` package, it looks like it needs a bit of work so I scoped that out of this PR.